sdk/communication/communication-tests-template.yml

```diff
@@ -31,3 +31,11 @@ stages:
             failIfCoverageEmpty: true
       EnvVars:
         SKIP_LIVE_TEST: TRUE
+      TestStepMavenInputs:
+        options: '$(DefaultOptions) -Dmaven.javadoc.skip=true -Dspotbugs.skip=true -Dcheckstyle.skip=true -Drevapi.skip=true -Djacoco.skip=$(SkipCoverage) -DtrimStackTrace=false -pl $(ProjectList)'
```

Comment:
Skip jacoco by cloud

Reply:
Are you overriding this entire set of inputs just to add the SkipCoverage option? If so we should make that customizable from the common instead of duplicating all these options.

Reply:
I agree that ideally we should make this customizable. My trouble is Communication would like to customize SkipCoverage on a per cloud base. It seems to me that can only be achieved with a variable rather than a parameter. Not sure how to make the parameter, TestStepMavenInputs, to work with a variable.

Reply:
I'm in the process of trying to normalize these options see my PR https://github.com/Azure/azure-sdk-for-java/pull/21797. The way I'd suggest is to define a variable in global.yml and set it to empty by default and if you want to override it to skip then you can set the variable to the correct value in your pipeline.